client/src/components/ManageAccount/ManageAccount.js

```diff
@@ -30,49 +31,61 @@ const ManageAccountView = () => {
     dispatch({ type: 'GET_PROFILE_DATA' });
   }, [dispatch, isLoading]);
   return (
-    <Container fluid className="manage-account-wrapper">
+    <div className="workbench-wrapper">
```

Comment:
__Why?__ Because there is no need to emulate `workbench-wrapper` with `manage-account-wrapper`; just use `workbench-wrapper`.